src/main/java/com/ericsson/gerrit/plugins/eiffel/configuration/EiffelPluginConfiguration.java

```diff
@@ -46,7 +62,71 @@ public EiffelPluginConfiguration(final String pluginName, final NameKey project,
             throw new ExceptionInInitializerError(String.format("Can't read %s plugin configuration for project %s: %s",
                     pluginName, project.toString(), e.getMessage()));
         }
```

Comment:
Since this method is now a bit bigger, could you break the try catch code into a method(loadPluginConfig(project, pluginName)).